### PR TITLE
Teams refactor

### PR DESCRIPTION
**Pull request by ahenzler**
Refactored Teams Manager

### CODE REVIEW

File: lib/teams_manager.rb [hunk 1]
```diff
@@ -16,36 +16,23 @@ def generate_list(data_path)
...
+  def count_of_teams
     teams.uniq.count
   end
 
-  def get_names_hash
+  def get_team_name(team_id)
     team_names_hash = {}
     @teams.each do |team|
       team_names_hash[team.team_id] = team.team_name
     end
```
Comment (denverdevelopments) [inline lib/teams_manager.rb:27]: Yes, makes a lot of sense to simply combine, and save one mostly unused method.

File: test/teams_manager_test.rb [hunk 2]
```diff
@@ -26,19 +26,8 @@ def test_count_of_teams_full_file
 
     assert_equal 32, team_manager.count_of_teams
   end
-
```
Comment (denverdevelopments) [inline test/teams_manager_test.rb:29]: Agreed, the test for that method that was phased out is no longer needed either.  Nice cleanup.

**Review by denverdevelopments — APPROVED** [review]

File: lib/game_teams_manager.rb [hunk 3]
```diff
@@ -37,35 +39,30 @@ def score_and_shots_by_team(season_games_ids)
...
-  def score_ratios_hash(season_games_ids)   ##Refactor?: 'hash' to 'accuracy'
-    hash = score_and_shots_by_team(season_games_ids)
-    hash.each do |team_id, pair|
-      ratio = calculate_ratios(pair)
-      hash[team_id] = ratio
-    end
-    hash
+  def score_ratios_hash(season_games_ids)
+    ratio = score_and_shots_by_team(season_games_ids)
+    ratio.transform_values {|pair| calculate_ratios(pair)}
```
Comment (ghost) [inline lib/game_teams_manager.rb]: love it

File: lib/teams_manager.rb (hunk 1 above)
Comment (ghost) [reply lib/teams_manager.rb:27]: This one is excellent.

**Review by ghost — APPROVED** [review]

**Pull request merged**